### PR TITLE
[ZH/EN] OWC2020 Grand Finals translation update, Fix typo in EN.

### PR DESCRIPTION
**Pull request by Kotoki1337**

### CODE REVIEW

**Review by TPGPL — APPROVED** [review]

File: wiki/Tournaments/OWC/2020/zh.md [hunk 1]
```diff
@@ -115,36 +115,55 @@ osu! 2020 世界杯由 osu! team 和多位社区成员举办。
...
-
-| A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
-| --: | :-- | :-: | :-: | :-: | :-: |
-| 美国 ![][flag_US] | ![][flag_CA] 加拿大 | **07:00 UTC+08** | 16:00 UTC-07 | 18:00 UTC-05 | ¹ |
-| 巴西 ![][flag_BR] | ![][flag_AU] 澳大利亚 | **09:00 UTC+08** | 22:00 (星期六) UTC-03 | 12:00 UTC+11 | ³ |
-| 日本 ![][flag_JP] | ![][flag_AU] 澳大利亚 | **18:00 UTC+08** | 19:00 UTC+09 | 21:00 UTC+11 | ³ |
-| 日本 ![][flag_JP] | ![][flag_DE] 德国 | **21:00 UTC+08** | 22:00 UTC+09 | 14:00 UTC+01 | ³ |
+| 加拿大 ![][flag_CA] | ![][flag_DE] 德国 | **02:00 UTC+08** | 13:00 (星期六) UTC-05 | 19:00 (星期六) UTC+01 | ¹ |
 
 ### 2020 年 11 月 30 日，星期一
```
Comment (WalterToro) [inline wiki/Tournaments/OWC/2020/zh.md:126]: This date should be updated.

Comment (Kotoki1337) [reply wiki/Tournaments/OWC/2020/zh.md:126]: Wait, what data? I'm not seeing the diff in en.

Comment (TPGPL) [reply wiki/Tournaments/OWC/2020/zh.md:126]: image

should  be 2020-12-06

Comment (Kotoki1337) [reply wiki/Tournaments/OWC/2020/zh.md:126]: > image
> 
> 
> 
> should  be 2020-12-06

Ohh my bad

Comment (Kotoki1337) [reply wiki/Tournaments/OWC/2020/zh.md:126]: #4547

**Pull request merged**